nat-lab/tests/telio.py

```diff
@@ -472,7 +472,15 @@ async def on_stderr(stderr: str) -> None:
 
                 print(datetime.now(), "Test cleanup: Shutting down")
                 if self._libtelio_proxy:
-                    await self.get_proxy().flush_logs()
+                    # flush_logs() is allowed to fail here:
+                    try:
+                        await self.get_proxy().flush_logs()
+                    except:
+                        print(
+                            datetime.now(),
+                            "Test cleanup: log flush failed",
+                        )
+
                     await self.get_proxy().shutdown(self._connection.target_name())
```

Comment:
I also think that shutdown should be inside `try` block, because if we get error on `flush_logs`, it is likely that we will get error on `shutdown`. Also would be nice if we could print what exception we got for more info.

Reply:
nvm, don't add shutdown, it is already handled :smile:, but print exception as mentioned before